### DISCUSSION TITLE
How to access each element of a hovered row in a ui.table?

### DISCUSSION THREAD

**Discussion by Ruibin-Liu** [zauberzeug/nicegui#993] [Ideas / Feature Requests]
I know [this discussion](https://github.com/zauberzeug/nicegui/discussions/774) has a solution to get a single row when clicking on it by changing the `selected` prop. But I want to access some elements of the row the user mouse is hovering upon. Ideally, it should invoke a callback function when such elements are hovered on.

**Comment by falkoschindler** [comment]
Thanks for the suggestion, @Ruibin-Liu!
As a workaround, you can introduce a custom event like in #774 :

```py
columns = [{'name': 'name', 'label': 'Name', 'field': 'name'}]
rows = [{'name': 'Alice'}, {'name': 'Bob'}, {'name': 'Carol'}]
table = ui.table(columns=columns, rows=rows, row_key='name')
table.add_slot('body-cell', r'''
    <q-td :props="props" @mouseover="$parent.$emit('mouse_over', props)">
        {{ props.value }}
    </q-td>
''')
table.on('mouse_over', lambda msg: ui.notify(msg['args']['row']))
```
